bubblewrap.c

```diff
@@ -1474,7 +1474,25 @@ setup_newroot (bool unshare_pid,
         case SETUP_MAKE_SYMLINK:
           assert (op->source != NULL);  /* guaranteed by the constructor */
           if (symlink (op->source, dest) != 0)
-            die_with_error ("Can't make symlink at %s", op->dest);
+            {
+              if (errno == EEXIST)
+                {
+                  char existing[PATH_MAX] = "";
+                  if (readlink (dest, existing, sizeof (existing)) < 0)
```

Comment:
`readlink`(2) does not guarantee to NUL terminate on success

Reply:
Simple enough to account for.